frontend/packages/ceph-storage-plugin/src/components/attached-devices-mode/lso-disk-inventory/disk-replacement-modal.tsx

```diff
@@ -109,6 +108,14 @@ const DiskReplacementAction = (props: DiskReplacementActionProps) => {
       <ModalTitle>Disk Replacement</ModalTitle>
       <ModalBody>
         <p>This action will start preparing the disk for replacement.</p>
+        {isRebalancing && alertsMap[diskName].status !== Status.Offline && (
+          <Alert
+            variant="warning"
+            className="co-alert"
+            title="Data rebalancing is in progress"
```

Comment:
Can we say the user to check the dashboard to see rebalancing status?

Something like this?
`Go check the [OCS Dashboard](url) to see rebalance status`

Reply:
@shirimordechay

Reply:
Sounds good, may be just hyperlinking the <progress> in "Data rebalancing is in progress".

Reply:
so let's keep the text as it is and add after that a link txt saying "see data resiliency status". how does that sound? we can review it with documentation.

Reply:
Sounds good!